meta/documents/user_guide/en/plugin_installation.md

```diff
@@ -0,0 +1,45 @@
+# Wayfair plugin: Installation instructions
+
+1. Login to the Plentymarkets system
+
+2. Go to **plugins >> plentyMarketplace**, which will use the system owner’s credentials to log you in on the Marketplace
+
+3. Click the search button  in the top-right corner of the page
+
+4. Enter “Wayfair” into the box and press enter
+
+5. If “Already purchased” appears on the page, skip to step 8
+
+6. Click the **go to checkout** button
+7. Complete the purchasing form
+    1. Accept terms and conditions
+    2. Click **order now**
+
+8. Return to the Plentymarkets system, and go to **Plugins >> Plugin Overview**.
+
+9. Find/Create a Plugin Set where the plugin is to be installed - the proper plugin set will be marked as **Linked to** your "shop."
+
+10. Un-check the **installed** checkbox

```

Comment:
I did not verify steps 10-20 as I did not want to mess up any of the dev environments. If there is a safe way for me to test this let me know and I'll run through these steps.

Reply:
Yeah don't do that. We need to use the new UX.